src/containers/Tenant/Diagnostics/TenantOverview/TenantOverviewTableLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'bem-cn-lite';
+
+import DataTable from '@gravity-ui/react-data-table';
+import type {DataTableProps, THEMES} from '@gravity-ui/react-data-table';
+
+import {
+    TENANT_OVERVIEW_TABLES_LIMIT,
+    TENANT_OVERVIEW_TABLES_SETTINGS,
+} from '../../../../utils/constants';
+import type {IResponseError} from '../../../../types/api/error';
+import {TableSkeleton} from '../../../../components/TableSkeleton/TableSkeleton';
+import {ResponseError} from '../../../../components/Errors/ResponseError';
+
+const b = cn('tenant-overview');
+
+interface TenantOverviewTableLayoutProps<T> extends Omit<DataTableProps<T>, 'theme'> {
+    theme?: THEMES | string;
```

Comment:
1. Why it's also a `string`?
2. I think this prop could be committed completely. We don't use tables with other themes